File: TD/TD2/Code/P1.cpp
/**
 * Vérifier si un entier est premier ou non
 */
#include <cmath>
#include <iostream>

int main() 
{
  int a;
  bool b;

  b = true;

  std::cout << "Donnes un entier : " << std::flush;
  std::cin >> a;

  for (int i = 2; i < sqrt(a); ++i)
    if (a % i == 0)
      b = false;

  if (b)
    std::cout << a << " est un nombre premier." << std::endl;
  else
    std::cout << a << " n'est pas un nombre premier." << std::endl;

  return 0;
}

File: TD/TD2/Code/Python/P1.py
#!/usr/bin/python
# Vérifier si un entier est premier ou non
from math import sqrt

b = True

print('Donne moi un entier : ', end='')
a = int(input())

for i in range(2, int(sqrt(a)) + 1, 1):
    if a % i == 0:
        b = False

if b:
    print(a, ' est un nombre premier.', end='')
else:
    print(a, 'n\'est pas un nombre premier.', end='')
